spec/unit/beaker/module_install_helper_spec.rb

```diff
@@ -1,12 +1,87 @@
 require 'spec_helper'
 
-describe 'ModuleInstallHelper' do
+describe Beaker::ModuleInstallHelper do
+  context 'hosts_to_install_module_on with split master/agent setup' do
+    let(:hosts) do
+      [
+        { 'roles' => %w(master database dashboard classifier) },
+        { 'roles' => ['agent'] }
+      ]
+    end
+
+    it 'returns a node with master role' do
+      expect(hosts_to_install_module_on.first['roles']).to include 'master'
+    end
+  end
+
+  context 'hosts_to_install_module_on with split master/agent setup' do
```

Comment:
The duplicated `hosts_to_install_module_on` in the context description hints at this and the previous context needing a joint parent. "with" is always a good indicator for denoting a context, and therefore usually should go first:

    context "#hosts_to_install_module_on" do
      context "with split master/agent setup" do ... end
      context "with solo agent setup" do ... end

PS: also the description here does not describe the reality of the test.